README.md

```diff
@@ -13,82 +13,142 @@ It's a debug only package, so it does not affect your production build.
+While spacejam requires mocha, meteor-coverage is agnostic on this point. If the `COVERAGE` flag is true, it instruments your app and provides an HTTP API to generate reports.
 
-For app, edit your `package.json` with the following
+These options are supported :
+
+    [run|test|test --full-app|test-packages]
+
+However, you need to run your app with the following driver package :  
+
+    [spacejam|meteor] --driver-package practicalmeteor:mocha-console-runner
+
+## Setup spacejam
+
+If you have any trouble, refer to this example of Meteor application [meteor-coverage-app-exemple](https://github.com/serut/meteor-coverage-app-exemple) to see how a test runner can execute yours tests, save coverage and send it to coveralls. Or feel free to open an issue. For now `serut/spacejam:windows-suppport-rc4` is only a fork but it will be merged someday.  
+
+
+Add the following dependencies in your `package.json`:
+
+    meteor npm init # If the package.json file does not exist 
+    meteor npm i --save-dev https://github.com/serut/spacejam/tarball/windows-suppport-rc4 
+
+Add what you need to run your app inside your `package.json`:
 
-    {
-    "devDependencies": {
-        [...]
-        "spacejam": "https://github.com/serut/spacejam/tarball/windows-suppport-rc4",
-        "coveralls": "^2.11.11",
-        "codecov.io": "^0.1.6"
-    },
-    {
     "scripts": {
-        [...]
-        "test-coverage-app-unit": "spacejam test                  --driver-package practicalmeteor:mocha-console-runner --coverage out_coverage",
-        "test-coverage-app-full": "spacejam test --full-app       --driver-package practicalmeteor:mocha-console-runner --coverage 'in_coverage|out_lcovonly'",
-        "test-coverage-packages-mocha": "spacejam test-packages   --driver-package practicalmeteor:mocha-console-runner --coverage out_lcovonly"
-    },
-    "devDependencies": {
-        [...]
-        "spacejam": "https://github.com/serut/spacejam/tarball/windows-suppport-rc4",
-        "coveralls": "^2.11.11",
-        "codecov.io": "^0.1.6"
-    },
+        [APP]
+        "test": "meteor npm run lint:fix & meteor npm run test:app-unit-coverage & ...",
+        "test:app-unit-coverage": "spacejam-mocha test            --coverage out_coverage",
+        "test:app-full-coverage": "spacejam-mocha test --full-app --coverage 'in_coverage|out_lcovonly'",
+        "test:packages-coverage": "spacejam-mocha test-packages   --coverage out_lcovonly"
+        
+        [PCKGS]
+        "test": "spacejam-mocha test-packages ./ --coverage out_lcovonly ",
+        "test:watch": "meteor npm run lint:fix & meteor npm run test:packages-coverage-watch",
+        "test:packages-coverage-watch": "meteor test-packages --driver-package practicalmeteor:mocha --settings settings.coverage.json",
```

Comment:
Just discovered that there exist a shorten form for `meteor test-packages --driver-package practicalmeteor:mocha ...` too: **`meteor-mocha ...`**. :metal: